wordless/helpers/render_helper.php

```diff
@@ -246,21 +248,30 @@ function wl_yield() {
         render_template($current_view, $current_locals);
     }
 
-    private function ensure_dir($dir) {
+    private function ensure_tmp_dir() : bool {
+        $tmpDir = Wordless::theme_temp_path();
+        $tmp_dir_exists_and_writable = $this->ensure_dir( $tmpDir );
+
+        return apply_filters('wordless_tmp_dir_exists', $tmpDir, $tmp_dir_exists_and_writable );
+    }
+
+    private function ensure_dir( $dir ) : bool {
+
+        $dir_exists_and_writable = false;
 
         if (!file_exists($dir)) {
             mkdir($dir, 0770);
         }
 
         if (!is_writable($dir)) {
-            chmod($dir, 0770);
+            chmod($dir, 770);
```

Comment:
mmm -_-.. maybe my ignorance... is it the same thing compared to 0770?

Reply:
Don't know: I'm not the author. Will check the docs.

Reply:
Permission uniformed to use 4 digits expression. TY